pom.xml

```diff
@@ -91,6 +91,12 @@
 	<repositories>
 
 		<repository>
+			<id>tychoM2E</id>
+			<layout>p2</layout>
+			<url>http://download.jboss.org/jbosstools/updates/requirements/m2eclipse-tycho/0.8.1.201704211436/</url>
```

Comment:
Why do you specify this repository in jbosstools-fuse/pom.xml? The repository should be defined in the appropriate JBTIS-TP, shouldn't it?